src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -1647,16 +1645,18 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
             >> Log.addContextDestructured "files" knownFiles
           )
 
-          let checker = (AVal.force checker)
+          cancelAllOpenFileCheckRequests ()
 
           for (file, aFile) in knownFiles do
             let (_, cts) = aFile |> AVal.force
 
             do!
-              forceTypeCheck checker file
+              forceTypeCheck file
               |> Async.withCancellationSafe (fun () -> cts.Token)
               |> Async.Ignore<unit option>
 
+          do! lspClient.CodeLensRefresh()
```

Comment:
Moves `CodeLensRefresh` to `DidSave` to avoid ionide from sending too many requests.